src/components/FloatingHelper/ClosablePopover/ClosablePopover.tsx

```diff
@@ -26,25 +28,32 @@ export interface ClosablePopoverOwnProps {
   onClose?: Function;
 }
 
+export enum Mode {
+  Hover = 'hover',
+  ClickToClose = 'click-to-close'
```

Comment:
isn't click enough?
`hover` & `click`

Reply:
I don't see a reason to abbreviate internal stuff, I'd rather be verbose.